concrete/controllers/single_page/dashboard/users/search.php

```diff
@@ -561,9 +564,24 @@ public function view($uID = false, $status = false)
     public function csv_export()
     {
         $search = $this->app->make('Concrete\Controller\Search\Users');
-        $searchResults = $search->getCurrentSearchObject();
-
-        $csvService = $this->app->make('helper/csv/user_list', [$searchResults, 'Users']);
-        $csvService->generate();
+        $result = $search->getCurrentSearchObject();
+        
+        $headers = [
+            'Content-Type' => 'text/csv',
+            'Content-Disposition' => 'attachment; filename=concrete5_users.csv'
```

Comment:
instead of hard-coding the file name, do we want to include the date in the filename? makes it easier for backup/archive purposes